script.js

```diff
@@ -189,3 +217,112 @@ function sortDes() {
   });
   showRecipesInGrid();
 }
+
+//*************************** Set cooking timer ******************************/
+
+let startTime = 0;
+let timerId;
+
+const startBtn = document.getElementById("startBtn");
+startBtn.addEventListener("click", startBtnClicked);
+const pauseBtn = document.getElementById("pauseBtn");
+pauseBtn.addEventListener("click", pauseBtnClicked);
+
+function recipeCookingTimeClicked(cookingTime) {
+  stopTimer();
+  const timerShow = document.getElementById("timeContainer");
+  timerShow.style.display = "flex";
+
+  document.getElementById("timeInput").value = cookingTime;
+  const displayTime = document.getElementById("timeDisplay");
+  let hrs = 0;
+  let mins = 0;
+  hrs = Math.floor(cookingTime / 60);
+  if (hrs < 10) {
+    hrs = "0" + hrs;
```

Comment:
You can have the browser do the formatting for you with this function:
```suggestion
    formatedHours = hrs.toLocaleString('en', { minimumIntegerDigits: 2 });
```

On a side note, avoid changing the type of your variables like this (`hrs` being a number and changed to a string). The browser allows it but it is a good habit to take so you won't be confused if you work with TypeScript or another typed programming language.